data/hazards.txt

```diff
@@ -100,3 +100,22 @@ hazard "Small Black Hole Gravity"
 		"damage dropoff" 120 1700
 		"dropoff modifier" 0
 
+
+
+hazard "Magnetar Radiation"
+	"constant strength"
+	"range" 3000
+	weapon
+		"discharge damage" 0.5
+		"damage dropoff" 120 3000
+		"hull damage" 1
+		"energy damage" 20
```

Comment:
Not sure how much sense hull damage makes here. How about disruption damage?

Reply:
I think that could work too